frameworks/hdfs/src/main/dist/svc.yml

```diff
@@ -18,7 +19,7 @@ pods:
     secrets:
       keytab:
         secret: {{SECURITY_KERBEROS_KEYTAB_SECRET}}
-        file: hdfs.keytab
+        file: secrets-hdfs.keytab
```

Comment:
Why was this renamed?

Reply:
+1 pls revert

Reply:
There was a difficulty in writing the output in the same file `hdfs.keytab` as the input, in the Java program which modifies the keytab file. That is why the name of input file was changed to `secrets-hdfs.keytab`. The output file name remains the same `hdfs.keytab`.

Reply:
The problem is that `hdfs.keytab` file has ownership of `root` user since it is a `secret`. `nobody` user cannot overwrite, move or copy this file. It has permissions `644`.
`rm: cannot remove 'hdfs.keytab': Device or resource busy`
I have added a comment against `secrets-hdfs.keytab` that it should be changed after Hdfs releases its bug fix.

Reply:
Okay that's a fair explanation. I don't think the comment directly reflects why it's being changed but I can live with it. We'll just have to address when there's a new release.